cmake/tribits/core/installation/TribitsProjectConfigTemplate.cmake.in

```diff
@@ -55,11 +55,11 @@ ${DEFINE_CMAKE_CURRENT_LIST_DIR_CODE_SNIPPET}
 ## Compilers used by ${PROJECT_NAME} build
 ## ---------------------------------------------------------------------------
 
-SET(${PROJECT_NAME}_CXX_COMPILER "${CMAKE_CXX_COMPILER}")
+SET(${PROJECT_NAME}_CXX_COMPILER "${CMAKE_CXX_COMPILER_FOR_CONFIG_FILE}")
 
-SET(${PROJECT_NAME}_C_COMPILER "${CMAKE_C_COMPILER}")
+SET(${PROJECT_NAME}_C_COMPILER "${CMAKE_C_COMPILER_FOR_CONFIG_FILE}")
 
-SET(${PROJECT_NAME}_Fortran_COMPILER "${CMAKE_Fortran_COMPILER}")
+SET(${PROJECT_NAME}_Fortran_COMPILER "${CMAKE_Fortran_COMPILER_FOR_CONFIG_FILE}")
```

Comment:
Where are `CMAKE_{CXX,C,Fortran}_COMPILER_FOR_CONFIG_FILE` defined?

Reply:
See the macro `TRIBITS_SET_COMPILER_VAR_FOR_CONFIG_FILE` defined at:

https://github.com/TriBITSPub/TriBITS/blob/master/tribits/core/package_arch/TribitsWriteClientExportFiles.cmake#L937

You can see where it gets called by just grepping that file.